dnf/cli/commands/install.py

```diff
@@ -74,15 +74,18 @@ def run(self, extcmds):
                 done = True
 
         # Install packages.
+        errs = []
         for pkg_spec in pkg_specs:
             try:
                 self.base.install(pkg_spec)
             except dnf.exceptions.MarkingError:
```

Comment:
Can you preserve the Error messages? Just duplicate 'No package %s%s%s available.' outputs - see what `yum install bla ble` or `dnf update bla ble` does. You can print it (using logger.info) directly when error occurs and hold the exception in variable, then raise the exception at the end of the cycle if there's any. You've understood and fixed the problem correctly but we want to keep the same UX (error message output).